composer.json

```diff
@@ -19,7 +19,7 @@
 
     "require": {
         "php":                      ">=5.3.9",
-        "symfony/framework-bundle": "~2.1",
+        "symfony/framework-bundle": "~2.3|~3.0",
```

Comment:
I'm not sure why version 2.1.* and 2.2.* are not supported anymore. But it would be nice to have version 3.0.* supported as this pull request is doing.